doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -225,6 +225,13 @@ nRF9160 samples
 
     * Increased the MCUboot partition size to the minimum necessary to allow bootloader FOTA.
 
+* :ref:`nrf_cloud_rest_device_message` sample:
+
+  * Added:
+
+    * Overlays to use the RTT instead of UART for testing purposes.
+    * New :c:func:`send_hello_world_msg` to create a message ID and send a Hello World message to NRF Cloud.
```

Comment:
```suggestion
    * New :c:func:`send_hello_world_msg` function to create a message ID and send a Hello World message to nRF Cloud.
```

Reply:
@tony-le-24  I don't think the function name is worth mentioning, especially since this is a sample.
As the commit message indicates, the relevant functional change is that the hello world message sent to nrf cloud now contains a timestamp (message id).